packages/website/src/components/AboutMentor/AboutMentor.tsx

```diff
@@ -0,0 +1,37 @@
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { VStack } from '@coderscamp/ui/components/Stack';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+
+import { MentorProfits } from './MentorProfits.data';
+import { ProfitDescription } from './ProfitDescription';
+
+export const AboutMentor = () => {
+  const titleSize = useBreakpointValue({ base: '4xl', sm: '6xl' } as const, 'base');
```

Comment:
💭 There is no need to use `useBreakpointValue` for the `size` property as it accepts responsive values by default